examples/examples_test.go

```diff
@@ -28,6 +28,9 @@ func getCwd(t *testing.T) string {
 }
 
 func getBaseOptions() integration.ProgramTestOptions {
```

Comment:
You need to pipe `testing.T` through where `getBaseOptions` is called as well.
```suggestion
func getBaseOptions(t *testing.T) integration.ProgramTestOptions {
```

Reply:
That's what I get for trying to commit code via github UI :pointy_haired_boss: